packages/firestore/test/integration/api_internal/bundle.test.ts

```diff
@@ -0,0 +1,309 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as firestore from '@firebase/firestore-types';
+import { expect } from 'chai';
+import {
+  apiDescribe,
+  toDataArray,
+  withAlternateTestDb,
+  withTestDb
+} from '../util/helpers';
+import { DatabaseId } from '../../../src/core/database_info';
+import { key } from '../../util/helpers';
+import { EventsAccumulator } from '../util/events_accumulator';
+import { TestBundleBuilder } from '../../unit/util/bundle_data';
+import { newTextEncoder } from '../../../src/platform/serializer';
+import { collectionReference } from '../../util/api_helpers';
+import { LoadBundleTask } from '../../../src/api/bundle';
+
+// TODO(b/162594908): Move this to api/ instead of api_internal.
```

Comment:
I would consider this blocking as this change will ensure that this feature works in the minified Browser build and is also required to ensure that the firestore-exp code you added is tested.

Reply:
Done.